### PR TITLE
Fix the formatting problem forever and ever.

### PR DESCRIPTION
**Pull request by XVincentX**
Connect #493 

Upgrading to the latest version of eslint let me run a format fix on the whole codebase in one single hit.

If we merge this, we can finally stop dealing with whitespaces and review real code.

Next step is prettier, but I need time to figure out what's the best way to introduce it without having to switch mental model.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +56,8 @@
       "node_modules",
       "coverage",
       "migrations",
-      "test"
+      "test",
+      "bin"
```
Comment (DrMegavolt) [inline package.json]: it is location of CLI tool, it should be checked

Comment (XVincentX) [reply package.json]: Ok, but Codeconv will not pass, as the formatting process has touched parts of the code that aren't tested (such as `bin`)

**Comment by XVincentX** [comment]
@DrMegavolt Done. We'll never see the big green here unfortunately, but it should be ready to go.

**Comment by XVincentX** [comment]
@DrMegavolt Rebased with current master. How do you feel about this monster?

**Pull request merged**